### PR TITLE
Syncing internal proto changes to opensource protos.

### PR DESCRIPTION
**Pull request by vtsao**

### CODE REVIEW

**Comment by pombredanne** [comment]
@vtsao Do you mind to explain a bit what this is about? 

You committed all this code in a single commit with no message?

Also did you check #65 before defining a format in https://github.com/Grafeas/Grafeas/pull/68/files#diff-f7adc668de6d65f4edcd73d223b55a32R244 ? That would be nice :)

**Comment by R2wenD2** [comment]
@pombredanne @vtsao is replacing the external source_context dependency with messages in our project. For now this is largely a drop-in replacement, which will give us flexibility to change the messages in the future.

File: v1alpha1/proto/grafeas.proto [hunk 1]
```diff
@@ -16,1093 +16,1352 @@ syntax = "proto3";
...
-    // The filter expression.
-    string filter = 2;
-    // Number of occurrences to return in the list.
-    int32 page_size = 3;
-    // Token to provide to skip to a particular spot in the list.
-    string page_token = 4;
+  // The name field contains the project Id. For example:
+  // "projects/{project_id}
+  // @Deprecated
+  string name = 1;
```
Comment (R2wenD2) [inline v1alpha1/proto/grafeas.proto]: I intentionally left fields we'd deprecated out of the oss proto. Please remove.

File: v1alpha1/proto/grafeas.proto [hunk 2]
```diff
@@ -16,1093 +16,1352 @@ syntax = "proto3";
...
-// Request to insert a new occurrence
+// Request to insert a new occurrence.
 message CreateOccurrenceRequest {
-    // This field contains the projectId for example: "projects/{project_id}"
-    string parent = 3;
-    // The occurrence to be inserted
-    Occurrence occurrence = 2;
+  // The name of the project.  Should be of the form "projects/{project_id}".
+  // @Deprecated
+  string name = 1;
```
Comment (R2wenD2) [inline v1alpha1/proto/grafeas.proto]: Please remove deprecated fields.

File: v1alpha1/proto/grafeas.proto [hunk 3]
```diff
@@ -16,1093 +16,1352 @@ syntax = "proto3";
...
-    // The filter expression.
-    string filter = 2;
-    // Number of notes to return in the list.
-    int32 page_size = 3;
-    // Token to provide to skip to a particular spot in the list.
-    string page_token = 4;
+  // The name field will contain the project Id for example:
+  // "providers/{provider_id}
+  // @Deprecated
+  string name = 1;
```
Comment (R2wenD2) [inline v1alpha1/proto/grafeas.proto]: remove deprecated field

File: v1alpha1/proto/grafeas.proto [hunk 4]
```diff
@@ -16,1093 +16,1352 @@ syntax = "proto3";
...
-    // "project/{project_id}
-    string parent = 4;
-    // The ID to use for this note.
-    string note_id = 2;
-    // The Note to be inserted
-    Note note = 3;
+  // The name of the project.
+  // Should be of the form "providers/{provider_id}".
+  // @Deprecated
+  string name = 1;
```
Comment (R2wenD2) [inline v1alpha1/proto/grafeas.proto]: Remove deprecated field.

File: v1alpha1/proto/grafeas.proto [hunk 5]
```diff
@@ -16,1093 +16,1352 @@ syntax = "proto3";
...
+
+  // This field contains the project Id for example:
+  // "project/{project_id}
```
Comment (R2wenD2) [inline v1alpha1/proto/grafeas.proto]: should be projects/{PROJECT_id}

File: v1alpha1/proto/grafeas.proto (hunk 1 above)
Comment (vtsao) [reply v1alpha1/proto/grafeas.proto]: Done.

File: v1alpha1/proto/grafeas.proto (hunk 2 above)
Comment (vtsao) [reply v1alpha1/proto/grafeas.proto]: Done.

File: v1alpha1/proto/grafeas.proto (hunk 4 above)
Comment (vtsao) [reply v1alpha1/proto/grafeas.proto]: Done.

File: v1alpha1/proto/grafeas.proto (hunk 5 above)
Comment (vtsao) [reply v1alpha1/proto/grafeas.proto]: Done.

**Comment by vbatts** [comment]
LGTM

**Pull request merged**